### DISCUSSION TITLE
Font rendering issue (Arch Linux)

### DISCUSSION THREAD

**Discussion by louwers** [kovidgoyal/kitty#4731] [Q&A]
I have enabled [sub-pixel rendering](https://wiki.archlinux.org/title/font_configuration#Subpixel_rendering) on Arch Linux. Now my fonts are not rendering correctly in Kitty:

![image](https://user-images.githubusercontent.com/649392/155234055-ea435b0e-40d8-41ab-9724-dbcb67ea6a0d.png)

I've read that Kitty does not support sub-pixel rendering on Linux.

Is there a way to disable sub-pixel rendering for just Kitty?

**Comment by kovidgoyal** [comment]
kitty doesnt do subpixelrendering. There is nothing to disable. From your screenshot it looks like a different font is being used than you expect. Use --debug-font-fallback to see which fonts kitty is using. And set the font elated options in kitty.conf accordingly.
